### PR TITLE
Quick fix due to the change of the site yggtorrent

### PR DESCRIPTION
**Pull request by PHD59fr**
Fixes #

Proposed changes in this pull request:
- Fix link to download on YggTorrent
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by PHD59fr** [comment]
Following the change of november 17th, 2017 on the providers YggTorrent, the format of the download links have changed, here the fix for this ;)

File: sickbeard/providers/yggtorrent.py [hunk 1]
```diff
@@ -123,7 +123,10 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                                 continue
 
                             title = cells[0].find('a', class_='torrent-name').get_text(strip=True)
-                            download_url = urljoin(self.url, cells[0].find('a', target='_blank')['href'])
+                            for download_img in cells[0].select('a[href] img'):                                   
+                                if download_img['src'] == "https://yggtorrent.com/static/icons/icon_download.gif":
+                                    download_url = urljoin(self.url, download_img.parent['href'])
+
```
Comment (miigotu) [inline sickbeard/providers/yggtorrent.py]: break the forloop if the url was found

Comment (PHD59fr) [reply sickbeard/providers/yggtorrent.py]: done

**Comment by ppicppic** [comment]
still doesn't help:

2017-11-28 08:41:17 WARNING::SEARCHQUEUE-BACKLOG-295683::[Yggtorrent]::Unable to connect to provider

2017-11-29 20:04:27 ERROR::SEARCHQUEUE-BACKLOG-284133::[Yggtorrent]::Failed parsing provider.
Traceback (most recent call last):
  File "/.sickrage/sickrage/providers/torrent/yggtorrent.py", line 131, in parse
    download_url = urljoin(self.urls['base_url'], cells[0].find('a', target='_blank')['href'])
TypeError: 'NoneType' object has no attribute '__getitem__'

**Comment by PHD59fr** [comment]
The provider has changed the address of the site www.yggtorrent.com to ww1.yggtorrent.com, I fix the problem tonight

**Comment by PHD59fr** [comment]
it's ok for me, I'm doing PR after:

09:17:36 INFO::SEARCHQUEUE-MANUAL-311196 :: Beginning manual search for: [Chance - S02E04 - The Coping Mechanism]
09:17:41 INFO::SEARCHQUEUE-MANUAL-311196 :: [Torrent9] :: Performing episode search for Chance
09:17:47 INFO::SEARCHQUEUE-MANUAL-311196 :: [Torrent9] :: Performing season pack search for Chance
09:17:47 INFO::SEARCHQUEUE-MANUAL-311196 :: [Cpasbien] :: Performing episode search for Chance
09:17:48 INFO::SEARCHQUEUE-MANUAL-311196 :: [YggTorrent] :: Performing episode search for Chance
09:17:50 INFO::SEARCHQUEUE-MANUAL-311196 :: [YggTorrent] :: Quality of Chance.S02E04.FRENCH.720p.HDTV.x264-SH0W.mkv is 720p HDTV
09:17:50 INFO::SEARCHQUEUE-MANUAL-311196 :: [YggTorrent] :: Quality of Chance.S02E04.FRENCH.720p.HDTV.x264-SH0W.mkv is 720p HDTV
09:17:50 INFO::SEARCHQUEUE-MANUAL-311196 :: Downloading Chance.S02E04.FRENCH.720p.HDTV.x264-SH0W.mkv from YggTorrent

**Pull request merged**